python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -223,7 +223,7 @@ def expand_class_scores(
 def _wrap_image_model(model, examples, model_task, is_function,
                       number_of_classes: int = None,
                       classes: Union[list, np.array] = None,
-                      device='cpu'):
+                      device="auto"):
```

Comment:
May recommend making an enum for `auto` considering this hard-coded string is being used in more than one places.